tests/src/FunctionalJavascript/Integration/AutosaveFormTest.php

```diff
@@ -57,7 +57,7 @@ public function testAutosaveInExistingEntity() {
     $this->pressRestoreButton();
     $this->assertEquals([5, '$ID:Tag2'], $page->findField('field_tags[]')->getValue());
     $this->assertEquals('New title', $page->findField('title[0][value]')->getValue());
-    $this->assertNotEmpty($page->find('css', '.form-item-field-paragraphs-4-subform-field-text-0-value'));
+    $this->assertNotEmpty($page->find('css', '.form-item-field-paragraphs-3-subform-field-text-0-value'));
```

Comment:
Since we removed the youtube paragraph from our testing content, here is one paragraph less